spec/lib/declare_schema/model/index_definition_spec.rb

```diff
@@ -133,6 +133,46 @@ class IndexDefinitionCompoundIndexModel < ActiveRecord::Base
 
         it { is_expected.to eq("users__last_name_first_name_middle_name") }
       end
+
+      context 'with long table name' do
+        let(:table_name2) { 'user_domains_extra' }
+        {
+          34 => '__last_name_first_name_middle_name',
+          35 => 'u__last_name_first_name_middle_name',
+          36 => 'u4__last_name_first_name_middle_name',
+          37 => 'us4__last_name_first_name_middle_name',
+          38 => 'us48__last_name_first_name_middle_name',
+          39 => 'use48__last_name_first_name_middle_name',
+          40 => 'use481__last_name_first_name_middle_name',
+          41 => 'user481__last_name_first_name_middle_name',
+          42 => 'user4814__last_name_first_name_middle_name',
+          43 => 'user_4814__last_name_first_name_middle_name',
+          44 => 'user_d4814__last_name_first_name_middle_name',
+          45 => 'user_do4814__last_name_first_name_middle_name',
+          46 => 'user_dom4814__last_name_first_name_middle_name',
+          47 => 'user_doma4814__last_name_first_name_middle_name',
+          48 => 'user_domai4814__last_name_first_name_middle_name',
+          49 => 'user_domain4814__last_name_first_name_middle_name',
+          50 => 'user_domains4814__last_name_first_name_middle_name',
+          51 => 'user_domains_4814__last_name_first_name_middle_name',
+          52 => 'user_domains_extra__last_name_first_name_middle_name',
+        }.each do |len, index_name|
```

Comment:
Nit: It'd be nice to also show the `len == 53` case here to prove that it behaves the same as `len == 52`. Not strictly necessary with the other tests but I like how it illustrates the point.

Reply:
Added: https://github.com/Invoca/declare_schema/pull/158/commits/772dd90cc86b668eb2f412da3e8a51a74e0ccf64